config/locales/components.en.yml

```
@@ -0,0 +1,5 @@
+en:
+  form_errors_component:
+    title:
+      one: '1 error occured while saving :'
+      other: '%{count} errors occured while saving :'
```

```suggestion
      one: '1 error occured while saving:'
      other: '%{count} errors occured while saving:'
```

There is no space before `:` in English.